typings/balena-sdk.d.ts

```diff
@@ -164,8 +164,11 @@ declare namespace BalenaSdk {
 		name: string;
 		type: string;
 		min?: number;
+		max?: number;
+		hidden?: boolean;
+		when?: Dictionary<number | string | boolean>;
```

Comment:
From what I could find in the `device-types/v1/` endpoint, this seems to always hold a string. Here are the values I could find on prod.
```
"coprocessorCore": "16",
"network": "wifi"
"os": "linux"
"os": "osx"
"os": "win32"
"processorCore": "quad"
"processorCore": "single"
"processorType": "Z7010"
"processorType": "Z7020"
```
I'm fine with it though.

Reply:
@thgreasi Things like `coprocessorCore` could easily be a number, and I am not sure how these are specified and validated. If there is no validation, it is safer to assume that the `when` condition can be any of them.